src/components/TextInput/index.native.tsx

```diff
@@ -2,10 +2,12 @@ import React, {forwardRef, useEffect} from 'react';
 import {AppState, Keyboard} from 'react-native';
 import useThemeStyles from '@styles/useThemeStyles';
 import BaseTextInput from './BaseTextInput';
-import * as baseTextInputPropTypes from './BaseTextInput/baseTextInputPropTypes';
+import type BaseTextInputProps from './BaseTextInput/types';
+import type {BaseTextInputRef} from './BaseTextInput/types';
```

Comment:
Maybe put it in one import

Reply:
I think we cannot do that 
<img width="783" alt="image" src="https://github.com/Expensify/App/assets/25230417/babfad0a-8bdb-49fe-b68e-94fba88bf843">
unless I will change export type to be named export not the default . @VickyStash @fabioh8010 what do you think?

Reply:
@kubabutkiewicz We have other places without default export [1](https://github.com/Expensify/App/blob/main/src/components/DraggableList/types.ts), [2](https://github.com/Expensify/App/blob/main/src/components/ErrorBoundary/types.ts), [3](https://github.com/Expensify/App/blob/main/src/components/AutoCompleteSuggestions/types.ts), so I think it should be ok